RevenueCatUI/Views/AsyncButton.swift

```diff
@@ -47,7 +47,7 @@ struct AsyncButton<Label>: View where Label: View {
             Button {
                 self.error = nil
             } label: {
-                Text("OK")
+                Text("OK", bundle: .module)
```

Comment:
Does `.module` work with cocoapods? Asking this if we decided to try and call `RevenueCatUI` views from React Native, Flutter, etc 🤷‍♂️

Reply:
Yes :) it’d be equivalent to `Bundle(forClass:)`

Reply:
But we'd have to manually add support for CocoaPods anyway, which we won't initially.